verifier/apps/main/src/main.rs

```diff
@@ -3,27 +3,34 @@ use global_utils::config_variant::ConfigVariant;
 use global_utils::env_parser::lookup_ip_addr;
 use global_utils::logger::init_logger;
 use persistent_storage::init::PostgresRepo;
-use tokio;
 use tokio::net::TcpListener;
 use tracing::instrument;
 use verifier_config_parser::config::ServerConfig;
 use verifier_local_db_store::PostgresDbCredentials;
+use verifier_utils::frost_signer::create_frost_signer;
 
 #[instrument(level = "debug", ret)]
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let _logger_guard = init_logger();
 
-    let app_config = ServerConfig::init_config(ConfigVariant::init())?;
-    let postgres_creds = PostgresDbCredentials::from_db_url()?;
+    let config_path = std::env::var("CONFIG_PATH").unwrap();
+
+    let app_config = ServerConfig::init_config(ConfigVariant::OnlyOneFilepath(config_path))?;
+    tracing::debug!("App config: {:?}", app_config);
+
+    let postgres_url = std::env::var("DATABASE_URL").unwrap();
+    let postgres_creds = PostgresDbCredentials { url: postgres_url };
```

Comment:
```suggestion
    let postgres_creds = PostgresDbCredentials::from_db_url();
```
it takes from `DATABASE_URL` env
https://github.com/distributed-lab/runes-spark-bridge/blob/895d75383baeb9aac382d5a5932069c15d2c35b5/crates/persistent_storage/src/config.rs#L24C23-L24C29